demisto_sdk/commands/init/contribution_converter.py

```diff
@@ -502,6 +502,29 @@ def create_contribution_items_version_note(self):
                     f"{item_versions.get('latest_version', '')}\n"
                 )
 
+            self.contribution_items_version_note += (
+                ">\n"
+                "> **For the Reviewer:**\n"
+                "> 1. Compare the code of this PR with the latest version of the pack, make sure to understand what are"
+                " the changes that the contributor intended to contribute, and **solve the conflicts in accordance**.\n"
+                "> 2. In case improvements are needed, instruct the contributor to edit the code through the **Github "
+                "Code-Space** and **Not through the XSOAR UI**.\n"
+            )
+
+            self.contribution_items_version_note += (
+                f">\n"
+                f"> **For the Contributor:**\n @{self.gh_user}\n"
+                f"> In case you are requested by your reviewer to improve the code or to make changes, please submit "
+                f"them through the **Github Code-Space** and **Not through the XSOAR UI**.\n"
```

Comment:
Let's change the terminology to `GitHub Codespaces`:
https://docs.github.com/en/codespaces/overview